packages/sdk/src/index.ts

```diff
@@ -243,3 +256,7 @@ export class Gomu {
     };
   }
 }
+
+function parseError(err: unknown): string {
```

Comment:
Should be `formatError` since we're returning a string here. `parseError` is parsing a string into an Error.